src/app/components/ResourceHints/index.jsx

```diff
@@ -1,26 +1,24 @@
 import React, { Fragment } from 'react';
+import { arrayOf, string } from 'prop-types';
 
-const ResourceHints = () => (
+const ResourceHints = ({ assetOrigins }) => (
   <Fragment>
-    <link
-      rel="preconnect"
-      href="https://ichef.bbci.co.uk"
-      crossOrigin="anonymous"
-    />
-    <link
-      rel="preconnect"
-      href="https://static.bbci.co.uk"
-      crossOrigin="anonymous"
-    />
-    <link
-      rel="preconnect"
-      href="https://gel.files.bbci.co.uk"
-      crossOrigin="anonymous"
-    />
-    <link rel="dns-prefetch" href="https://ichef.bbci.co.uk" />
-    <link rel="dns-prefetch" href="https://static.bbci.co.uk" />
-    <link rel="dns-prefetch" href="https://gel.files.bbci.co.uk" />
+    {assetOrigins.map(origin => (
+      <link
+        rel="preconnect"
+        key={origin}
+        href={origin}
+        crossOrigin="anonymous"
+      />
+    ))}
+    {assetOrigins.map(origin => (
+      <link rel="dns-prefetch" key={origin} href={origin} />
```

Comment:
Probably best to use `nanoid` here, rather than `origin` as the key

Reply:
As discussed I don't think using `nanoid` is a good idea, my understanding is it doesn't produce the stable identifier that the `key` attribute expects.